packages/celotool/README.md

```diff
@@ -1,5 +1,7 @@
 # Celotool
 
+> This package is deprecated and should be removed
```

Comment:
We've never had this conversation, it's likely that this is true but I don't know, I still use celotool for a couple of things, and Javi may be using it too

Reply:
No one from devops is using it.
I think you and @pahor167 used it for port forwarding, but this use case is outdated now.
Please refer to manual approach defined in devops slack channel for port forwarding 😁 
Resolving with leaving note for future deprecation and removal 😌